### ISSUE TITLE
`coroutineScope` does not work on compose multiplatform `web` target

### ISSUE THREAD

**Issue by JoelKanyi** [adrielcafe/voyager#240]
Currently it working fine on Android, iOS and desktop after adding (`org.jetbrains.kotlinx:kotlinx-coroutines-swing`)

But when it comes to web, I'm getting this in one of my ScreenModels:

```
ERROR
Could not create instance for '[Singleton:'HomeScreenModel']'
InstanceCreationException: Could not create instance for '[Singleton:'HomeScreenModel']'
    at protoOf.create_jnbvc8_k$ (webpack-internal:///./kotlin/core-koin-core.js:829:15)
    at protoOf.create_jnbvc8_k$ (webpack-internal:///./kotlin/core-koin-core.js:996:55)
    at eval (webpack-internal:///./kotlin/core-koin-core.js:969:33)
    at protoOf.synchronized_5h0e7m_k$ (webpack-internal:///./kotlin/core-koin-core.js:2464:12)
    at protoOf.get_kchvk2_k$ (webpack-internal:///./kotlin/core-koin-core.js:1004:9)
    at protoOf.resolveInstance_cs8p06_k$ (webpack-internal:///./kotlin/core-koin-core.js:1602:70)
    at resolveValue (webpack-internal:///./kotlin/core-koin-core.js:1883:73)
    at resolveInstance (webpack-internal:///./kotlin/core-koin-core.js:1875:17)
    at protoOf.get_ocvqe2_k$ (webpack-internal:///./kotlin/core-koin-core.js:2134:13)
    at HomeScreen$composable (webpack-internal:///./kotlin/FocusBloom-shared.js:19728:30)
   ```
   
   
   When I remove any trace of the `coroutineScope` on the screenmodel, the web app seems to run.
   
   
   This a sample code snippet of how I'm using it:
   ```kotlin
   fun updateTask(task: Task) {
        screenModelScope.launch {
            tasksRepository.updateTask(task)
        }
    }
   ```
   
 I am using:
 - voyager = "1.0.0-rc07"
 - kotlin = "1.9.10"
 - compose = "1.5.10-beta01"

**Comment by JoelKanyi** [comment]
Hey @DevSrSouza is this fix in `voyager = "1.0.0-rc08"`

**Comment by DevSrSouza** [comment]
No, sorry, it will be fixed in 1.0.0-rc09
